frontend/src/components/resourceMap/GraphView.tsx

```diff
@@ -129,7 +129,7 @@ function GraphViewContent({
     if (hasErrorsFilter) {
       filters.push({ type: 'hasErrors' });
     }
-    if (namespaces) {
+    if (namespaces && namespaces.size > 0) {
```

Comment:
This can be simplified:
```suggestion
    if (namespaces?.size > 0) {
```